openstack-installer/roles/os_neutron/defaults/main.yml

```diff
@@ -13,6 +13,8 @@ neutron_syslog_use: "{{ syslog_use | default(False) }}"
 
 neutron_debug: False
 
+neutron_physical_interface_driver: linuxbridge
+
 nova_admin_user: nova
```

Comment:
Please add this to os_nova/defaults/main.yml, too, otherwise ansible will abort on an undefined variable templating out nova.conf.
Otherwise the patch looks very good, thanks!
And my last wish: please squash it to one commit, a tutorial how to do it easily is here:
http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit